src/Backend/Modules/Blog/DependencyInjection/BlogExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Backend\Modules\Blog\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\Config\FileLocator;
+
+class BlogExtension extends Extension implements PrependExtensionInterface
+{
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        // no services to load, so an empty method
+    }
+
+    public function prepend(ContainerBuilder $container)
```

Comment:
you can add : void after the brackets

Reply:
fixed in 183dde936c84bcfa2ac666bd0fd4db7d5a2bb1dc